addons/purchase/migrations/9.0.1.2/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,186 @@
+---Fields in module 'purchase'---
+purchase     / account.invoice          / purchase_id (many2one)        : NEW relation: purchase.order
+#Nothing To Do : New Functionality added.
+
+purchase     / product.category         / property_account_creditor_price_difference_categ (many2one): NEW relation: account.account
+purchase     / product.template         / property_account_creditor_price_difference (many2one): NEW relation: account.account
+#Nothing To Do : New Functionality added.
+
+purchase     / product.template         / purchase_method (selection)   : NEW selection_keys: ['purchase', 'receive']
+#Nothing To Do : New Functionality added, default is receive (which will be automatically set to all products)
+
+purchase     / purchase.order           / bid_date (date)               : DEL 
+purchase     / purchase.order           / bid_validity (date)           : DEL 
+#Nothing To Do : Functionality removed now no longer needed.
+
+purchase     / purchase.order           / fiscal_position (many2one)    : was renamed to fiscal_position_id [nothing to to]
+#Nothing To Do : ORM will change the name automatically.
+
+purchase     / purchase.order           / group_id (many2one)           : NEW relation: procurement.group
+#Nothing To Do : New Functionality added.
+
+purchase     / purchase.order           / invoice_ids (many2many)       : DEL relation: account.invoice
+#Nothing To Do : Functionality removed now no longer needed.
+
+purchase     / purchase.order           / invoice_method (selection)    : DEL required: required, selection_keys: ['manual', 'order', 'picking'], req_default: order
+purchase     / purchase.order           / journal_id (many2one)         : DEL relation: account.journal
+#Nothing To Do : Functionality removed now no longer needed.
+
+purchase     / purchase.order           / location_id (many2one)        : DEL relation: stock.location, required: required
+#Nothing To Do : As picking type with default destination is already there.
+
+purchase     / purchase.order           / message_follower_ids (many2many): not a function anymore
+purchase     / purchase.order           / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+purchase     / purchase.order           / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# TODO : ???  # Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
```

Comment:
It seems to be managed in mail module. ref : sale migration module : 
https://github.com/OCA/OpenUpgrade/blob/9.0/addons/sale/migrations/9.0.1.0/openupgrade_analysis_work.txt#L39